src/Sulu/Bundle/SecurityBundle/Entity/BaseRole.php

```diff
@@ -57,6 +57,20 @@
     private $securityType;
 
     /**
+     * Update the timestamps (changed + created).
+     * This method is mapped to the PRE_PERSIST and PRE_UPDATE
+     * lifecycle events.
+     */
+    public function updateTimestamps()
```

Comment:
So do we revert this? I say lets just merge this and I will refactor the whole timestamp thing this week